lib/teleterm/clusters/cluster_servers.go

```diff
@@ -68,3 +72,84 @@ func (c *Cluster) GetServers(ctx context.Context) ([]Server, error) {
 
 	return results, nil
 }
+
+func (c *Cluster) GetServers(ctx context.Context, r *api.GetServersRequest) (*GetServersResponse, error) {
+	var (
+		clusterServers []types.Server
+		resp           *types.ListResourcesResponse
+		authClient     auth.ClientI
+		proxyClient    *client.ProxyClient
+		sortBy         types.SortBy
+		err            error
+	)
+
+	err = addMetadataToRetryableError(ctx, func() error {
+		proxyClient, err = c.clusterClient.ConnectToProxy(ctx)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		defer proxyClient.Close()
+		return nil
+	})
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	defer authClient.Close()
```

Comment:
I think it would be best to add this defer after a value is bound to `authClient`, just like we do it for `proxyClient`.

Reply:
Whoops, this was a typo. thank you